packages/office-ui-fabric-react/src/components/Layer/Layer.Props.ts

```diff
@@ -3,4 +3,19 @@ import * as React from 'react';
 export interface ILayerProps extends React.HTMLProps<HTMLDivElement> {
   /** Callback for when the layer is mounted. */
   onLayerMounted?: () => void;
-}
\ No newline at end of file
+
+  /**
+   * Callback for when the layer is mounted.
+   */
+  onLayerDidMount?: () => void;
```

Comment:
Tentative suggestion: If you're going this route you might want to add a few other life cycle options, like onLayerWillMount().